test/unit/read_preference.test.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const { ReadPreference } = require('../../src');
+const chai = require('chai');
+chai.use(require('chai-subset'));
+const expect = chai.expect;
+
+describe('ReadPreference', function () {
```

Comment:
```suggestion
describe('class ReadPreference', function () {
```
If you use the mocha explorer then you may notice that the buttons appear on the import at the top of the file instead of here, I have worked around this by changing up the naming a bit. Take a look at `test/unit/utils.test.ts` for some examples. It's a silly-ish work around but I think its worth it for the developer UX 😄